src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -48,13 +52,22 @@ public void setPersons(List<Person> persons) {
         this.persons.setPersons(persons);
     }
 
+    /**
+     * Replaces the contents of the tutorialTags list with {@code tutorialTags}.
```

Comment:
Same here. It is not a must to change but would be good in case we are generating help text with the javadoc comments.
Other instances of the same issue will be ignored in my review.